schemas/dbt/manifest/v12.json

```diff
@@ -17930,26 +18188,7 @@
                       "type": "string"
                     },
                     "resource_type": {
```

Comment:
This change also seems unrelated to my PR.

Reply:
I think this is a fix. Basically, the `resource_type` of a saved query isn't actually an enum, it's should _always_ be `saved_query`

Reply:
I think what's happening is we aren't running the `scripts/collect-artifact-schema.py` as often as we should. I'll bring this up to the rest of the core team. Maybe we can have a CI check for it.